nmodl/ode.py

```diff
@@ -12,7 +12,10 @@
 # import known_functions through low-level mechanism because the ccode
 # module is overwritten in sympy and contents of that submodule cannot be
 # accessed through regular imports
-known_functions = import_module('sympy.printing.ccode').known_functions_C99
+if int(sp.__version__.replace('.','')) < 17:
+    known_functions = import_module('sympy.printing.ccode').known_functions_C99
```

Comment:
versions like `1.6.1` like possible?

Reply:
yes, I noticed when I woke up this morning. Corrected